metis-indexing/src/main/java/eu/europeana/indexing/common/fullbean/FieldInputUtils.java

```diff
@@ -190,6 +213,52 @@ static String getResourceString(ResourceType obj) {
     return array.length > 0 ? array[0] : null;
   }
 
+  /**
+   * Gets literal value string.
+   *
+   * @param obj the LiteralType object
+   * @return a string from the LiteralType object
+   */
+  static String getLiteralValueString(LiteralType obj) {
+    return Optional.ofNullable(obj)
+                   .map(LiteralType::getString)
+                   .orElse(null);
```

Comment:
For consistency, I'd advice implementing this similarly to `getResourceString()`.